### ISSUE TITLE
ErrorSchemaValidation

### ISSUE THREAD

**Issue by Harmlos** [ecederstrand/exchangelib#317]
Hello

while try simple script:

```python
from exchangelib import DELEGATE, Account, Credentials
creds = Credentials(
    username='mydomain\myuser',·
    password='xxx')
account = Account(
    primary_smtp_address='myuser@mydomain.ru',
    credentials=creds,·
    autodiscover=True,·
    access_type=DELEGATE)
```

got error:

```python
No handlers could be found for logger "exchangelib.services"
Traceback (most recent call last):
  File "get_xml_from_mail.py", line 30, in <module>
    access_type=DELEGATE)
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/account.py", line 91, in __init__
    self.root = Root.get_distinguished(account=self)
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/folders.py", line 539, in get_distinguished
    folders = list(cls.get_folders(account=account, ids=[cls(account=account, name=cls.DISTINGUISHED_FOLDER_ID)]))
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/folders.py", line 530, in get_folders
    shape=IdOnly
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/services.py", line 887, in call
    shape=shape,
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/services.py", line 76, in _get_elements
    response = self._get_response_xml(payload=payload)
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/services.py", line 148, in _get_response_xml
    res = self._get_soap_payload(soap_response=soap_response_payload)
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/services.py", line 192, in _get_soap_payload
    cls._raise_soap_errors(fault=fault)  # Will throw SOAPError or custom EWS error
  File "/usr/local/lib/python2.7/dist-packages/exchangelib/services.py", line 215, in _raise_soap_errors
    raise vars(errors)[code](msg)
exchangelib.errors.ErrorSchemaValidation: The request failed schema validation.
```

In debug found string:

Request data: 
```xml
<?xml version="1.0" encoding="utf-8"?>
<s:Envelope xmlns:m="http://schemas.microsoft.com/exchange/services/2006/messages"
            xmlns:s="http://schemas.xmlsoap.org/soap/envelope/"
            xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types">
    <s:Header>
        <t:RequestServerVersion Version="Exchange2010_SP2"/>
        <t:TimeZoneContext>
            <t:TimeZoneDefinition Id="Russian Standard Time"/>
        </t:TimeZoneContext>
    </s:Header>
    <s:Body>
        <m:GetFolder>
            <m:FolderShape>
                <t:BaseShape>IdOnly</t:BaseShape>
                <t:AdditionalProperties>
                    <t:FieldURI FieldURI="folder:ChildFolderCount"/>
                    <t:FieldURI FieldURI="folder:EffectiveRights"/>
                    <t:FieldURI FieldURI="folder:FolderClass"/>
                    <t:FieldURI FieldURI="folder:DisplayName"/>
                    <t:FieldURI FieldURI="folder:ParentFolderId"/>
                    <t:FieldURI FieldURI="folder:TotalCount"/>
                    <t:FieldURI FieldURI="folder:UnreadCount"/>
                </t:AdditionalProperties>
            </m:FolderShape>
            <m:FolderIds>
                <t:DistinguishedFolderId Id="root">
                    <t:Mailbox>
                        <t:EmailAddress>userlogin@mydomain.ru</t:EmailAddress>
                        <t:MailboxType>Mailbox</t:MailboxType>
                        <t:RoutingType>SMTP</t:RoutingType>
                    </t:Mailbox>
                </t:DistinguishedFolderId>
            </m:FolderIds>
        </m:GetFolder>
    </s:Body>
</s:Envelope>
```

Response data:

```xml
<?xml version="1.0" encoding="utf-8"?>
<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/">
    <s:Body>
        <s:Fault>
            <faultcode xmlns:a="http://schemas.microsoft.com/exchange/services/2006/types">a:ErrorSchemaValidation
            </faultcode>
            <faultstring xml:lang="ru-RU">The request failed schema validation: The element 'Mailbox' in namespace
                'http://schemas.microsoft.com/exchange/services/2006/types' has invalid child element 'RoutingType' in
                namespace 'http://schemas.microsoft.com/exchange/services/2006/types'. List of possible elements
                expected: 'ItemId' in namespace 'http://schemas.microsoft.com/exchange/services/2006/types'.
            </faultstring>
            <detail>
                <e:ResponseCode xmlns:e="http://schemas.microsoft.com/exchange/services/2006/errors">
                    ErrorSchemaValidation
                </e:ResponseCode>
                <e:Message xmlns:e="http://schemas.microsoft.com/exchange/services/2006/errors">The request failed
                    schema validation.
                </e:Message>
                <t:MessageXml xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types">
                    <t:LineNumber>1</t:LineNumber>
                    <t:LinePosition>1009</t:LinePosition>
                    <t:Violation>The element 'Mailbox' in namespace
                        'http://schemas.microsoft.com/exchange/services/2006/types' has invalid child element
                        'RoutingType' in namespace 'http://schemas.microsoft.com/exchange/services/2006/types'. List of
                        possible elements expected: 'ItemId' in namespace
                        'http://schemas.microsoft.com/exchange/services/2006/types'.
                    </t:Violation>
                </t:MessageXml>
            </detail>
        </s:Fault>
    </s:Body>
</s:Envelope>
```

Full log in file

[new.txt](https://github.com/ecederstrand/exchangelib/files/1555898/new.txt)

**Comment by ecederstrand** [comment]
I believe this was fixed recently in https://github.com/ecederstrand/exchangelib/commit/a4dbbab63d62994bcb4e32441e3ad99e6b4b7539

**Comment by Harmlos** [comment]
yeah
install from dev branch work fine
before that i try to install only from pip library because work build was too fresh (2017-11-29)

**Comment by ecederstrand** [comment]
I just released v1.10.7 which contains the fix.
